src/editors/containers/ProblemEditor/components/EditProblemView/AnswerWidget/hooks.js

```diff
@@ -2,14 +2,30 @@ import { useState, useEffect } from 'react';
 import { StrictDict } from '../../../../../utils';
 import * as module from './hooks';
 import { actions } from '../../../../../data/redux';
-import { ProblemTypeKeys } from '../../../../../data/constants/problem';
+import { ProblemTypeKeys, RichTextProblems } from '../../../../../data/constants/problem';
 import { fetchEditorContent } from '../hooks';
 
 export const state = StrictDict({
   isFeedbackVisible: (val) => useState(val),
 });
 
-export const removeAnswer = ({ answer, dispatch }) => () => {
+export const removeAnswer = ({
+  answers,
+  answer,
+  problemType,
+  dispatch,
+}) => () => {
+  if (RichTextProblems.includes(problemType)) {
+    const currentAnswerTitles = fetchEditorContent({ format: 'text' }).answers;
```

Comment:
```suggestion
    const currentAnswerTitles = fetchEditorContent({ format: '' }).answers;
```

Reply:
giving it the parameter text converts it from rich text to plain text. when saving it to the redux we should maintain the rich text